### PR TITLE
PyQt5 app with twilio

### PR DESCRIPTION
**Pull request by MCodez**
A simple and robust PyQt5 application which sends cricket news directly to your mobile using Twilio. This is a best basic example to implement Twilio with other APIs or Libraries with Python3.

### CODE REVIEW

File: examples/PyCricket v1.0/cricketscore.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+for match in matches:
+    if('IND ' in match['mchdesc']):
+        print(match['id'])
+
+#Get the feed for your team     
+print()   
+finalscorecard=""
+for match in matches:
+    if 'IND vs PAK' in match['mchdesc']:
```
Comment (Kerl1310) [inline examples/PyCricket v1.0/cricketscore.py]: If this is getting the score for one team only, regardless of opponent, then this should be something like `if 'IND' in match['mchdesc']:`

File: examples/PyCricket v1.0/cricketscore.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    print(livematches[i])        
+      
+# Get your country match by this search 
+
+for match in matches:
+    if('IND ' in match['mchdesc']):
+        print(match['id'])
+
+#Get the feed for your team     
+print()
```
Comment (Kerl1310) [inline examples/PyCricket v1.0/cricketscore.py]: Unnecessary code

File: examples/PyCricket v1.0/cricketscore.py [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+        finalscorecard=finalscorecard+str(match['srs'])
+        finalscorecard=finalscorecard+"\n"+str(match['mchdesc'])
+        finalscorecard=finalscorecard+"\n"+str(match['type'])+str(match['mnum'])
+        finalscorecard=finalscorecard+"\n"+str(match['status'])
+        finalscorecard=finalscorecard+"\n"+str(match['mchstate'])
+        scorecard=c.scorecard(identity)
+        seperator="-------------------------------------------------------------------------------"
+        finalscorecard=finalscorecard+"\n"+seperator
+        finalscorecard=finalscorecard+"\n"+"BATTING TEAM :"+str(scorecard['scorecard'][0]['batteam'])
+        for i in range(len(scorecard['scorecard'][0]['batcard'])):
```
Comment (Kerl1310) [inline examples/PyCricket v1.0/cricketscore.py]: This should be split up and can be done without `range(len(`

File: examples/PyCricket v1.0/cricketscore.py [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+            temp=""
+            temp=str(scorecard['scorecard'][0]['batcard'][i]['name'])+" "+str(scorecard['scorecard'][0]['batcard'][i]['runs'])+"of"+str(scorecard['scorecard'][0]['batcard'][i]['balls'])+",  Fours :"+str(scorecard['scorecard'][0]['batcard'][i]['fours'])+", Sixes :"+str(scorecard['scorecard'][0]['batcard'][i]['six'])+", Dismissal :"+str(scorecard['scorecard'][0]['batcard'][i]['dismissal'])
```
Comment (Kerl1310) [inline examples/PyCricket v1.0/cricketscore.py]: This should be separated out and should make use of string interpolation.

File: examples/PyCricket v1.0/cricketscore.py [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+            temp=str(scorecard['scorecard'][0]['bowlcard'][i]['name'])+"Overs :"+str(scorecard['scorecard'][0]['bowlcard'][i]['overs'])+", Runs :"+str(scorecard['scorecard'][0]['bowlcard'][i]['runs'])+", Wickets :"+str(scorecard['scorecard'][0]['bowlcard'][i]['wickets'])
+            finalscorecard=finalscorecard+"\n"+temp
+
+print(finalscorecard)
+print(seperator)
+# Listing Matches affected by Rain
+print("Matches affected by Rain: ")
+
+for match in matches:
+    if(match['mchstate']=='rain'):
```
Comment (Kerl1310) [inline examples/PyCricket v1.0/cricketscore.py]: Should use `.lower()` to ensure correct casing, same for line 17

File: examples/PyCricket v1.0/cricketscore.py [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+        identity=match['id']
+        finalscorecard=finalscorecard+str(match['srs'])
+        finalscorecard=finalscorecard+"\n"+str(match['mchdesc'])
+        finalscorecard=finalscorecard+"\n"+str(match['type'])+str(match['mnum'])
+        finalscorecard=finalscorecard+"\n"+str(match['status'])
+        finalscorecard=finalscorecard+"\n"+str(match['mchstate'])
+        scorecard=c.scorecard(identity)
+        seperator="-------------------------------------------------------------------------------"
+        finalscorecard=finalscorecard+"\n"+seperator
+        finalscorecard=finalscorecard+"\n"+"BATTING TEAM :"+str(scorecard['scorecard'][0]['batteam'])
```
Comment (Kerl1310) [inline examples/PyCricket v1.0/cricketscore.py]: Create variables for the following to reduce excessive code and improve performance:
`scorecard['scorecard'][0]`,
`scorecard['scorecard'][0]['batteam']`
`scorecard['scorecard'][0]['batcard']`

File: examples/PyCricket v1.0/cricketscore.py [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+from pycricbuzz import Cricbuzz
+from twilio.rest import Client
+c = Cricbuzz() 
+matches = c.matches()
+message=""
+livematches=[]
+
+# printing live matches at that time
+for match in matches:
+    if(match['mchstate']=='inprogress'):
```
Comment (Kerl1310) [inline examples/PyCricket v1.0/cricketscore.py]: Add spaces around operators to better conform to PEP-8 style guidelines

File: examples/PyCricket v1.0/cricketscore.py [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+        livematches.append(match['mchdesc'])
+print("Live Matches :")       
+for i in range(len(livematches)):
+    print(livematches[i])
```
Comment (Kerl1310) [inline examples/PyCricket v1.0/cricketscore.py]: snake_case variable names to better conform to PEP-8 style guidelines

**Comment by eshanholtz** [comment]
While this is a very robust example, I'm not convinced its presence in this repository adds a lot of value for users. This seems more focused on PyQt5 than it does on using twilio-python generally. Closing this PR.

**Pull request closed without merging**